sharepoint/sharepoint-ps/sharepoint-online/Start-SPOSiteOpticalCharacterRecognitionBackfill.md

```diff
@@ -0,0 +1,76 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/Start-SPOSiteOpticalCharacterRecognitionBackfill
+applicable: SharePoint Online
+title: Start-SPOSiteOpticalCharacterRecognitionBackfill
+schema: 2.0.0
+author: FarreltinF
+ms.author: fanyi
+ms.reviewer:
+---
+
+# Start-SPOSiteOpticalCharacterRecognitionBackfill
+
+## SYNOPSIS
+
+> [!important]
+> Optical Character Recognition (OCR) is a pay-as-you-go feature, triggering this cmdlet will incur cost for your organization
```

Comment:
The comma is misplaced -- maybe replace it with "and" or "." instead. Also add `.` at end of sentence.